action/protocol/staking/contractstake_bucket_type.go

```diff
@@ -38,6 +38,15 @@ func (bt *ContractStakingBucketType) Deserialize(b []byte) error {
 	return bt.loadProto(&m)
 }
 
+// Clone clones the bucket type
+func (bt *ContractStakingBucketType) Clone() *ContractStakingBucketType {
```

Comment:
`clone`? no need to export

Reply:
`ContractStakingBucketType` is defined in `staking` and should be exported to access in contractstaking